.env.template

```diff
@@ -17,27 +17,27 @@ MINIO_API_URL=api.minio.cuahsi.io
 
 VITE_APP_NAME=discover
 
-# if using npm run dev, set VITE_APP_ORIGIN=http://localhost:5173
-VITE_APP_ORIGIN=https://localhost
+VITE_APP_ORIGIN=http://localhost:8080
 
 # for nested static deployment, set VITE_APP_BASE=/discover/
 VITE_APP_BASE=/
 
 # VITE_APP_FULL_URL=${VITE_APP_ORIGIN}${VITE_APP_BASE}
-VITE_APP_FULL_URL=https://localhost/
+VITE_APP_FULL_URL=http://localhost:8080/
 
 VITE_APP_API_HOST=localhost
 
 # VITE_APP_API_URL=https://${VITE_APP_API_HOST}/api
-VITE_APP_API_URL=https://localhost/api
+# currently the api route is a bit funky -- we will wait to modify this until pubsub-update branch is merged
+VITE_APP_API_URL=https://localhost/api/api
 
 # ALLOW_ORIGINS=${VITE_APP_ORIGIN}
-ALLOW_ORIGINS=https://localhost
+ALLOW_ORIGINS=http://localhost:8080
 
 # OAUTH2_REDIRECT_URL=${VITE_APP_API_URL}/auth/cuahsi/callback
-OAUTH2_REDIRECT_URL=https://localhost/api/auth/cuahsi/callback
+OAUTH2_REDIRECT_URL=https://localhost/api/api/auth/cuahsi/callback
```

Comment:
I didn't spend too much time cleaning up the `/api/api` nested routing because there would be conflicts with Scott's [pubsub-update](https://github.com/hydroshare/discovery-atlas/tree/pubsub-update) work. We can fix the nginx config and fast api routing after merging Scott's work later